test/integration/adapters/postmark_test.exs

```diff
@@ -5,23 +5,42 @@ defmodule Swoosh.Integration.Adapters.PostmarkTest do
 
   @moduletag integration: true
 
-  setup_all do
-    config = [api_key: System.get_env("POSTMARK_API_KEY"), domain: System.get_env("POSTMARK_DOMAIN")]
-    {:ok, config: config}
-  end
+  @config [
+    api_key: System.get_env("POSTMARK_API_KEY"),
+    domain:  System.get_env("POSTMARK_DOMAIN"),
+  ]
+
+  @base_email new
```

Comment:
See comment about compile-time/runtime above. This would be nicer as a function.